src/create_disk.sh

```diff
@@ -104,26 +104,53 @@ if [ "$arch" == "x86_64" ]; then
 		--boot-directory rootfs/boot \
 		$disk
 	ext="X64"
-else
+elif [ "$arch" == "aarch64" ]; then
 	mkdir -p rootfs/boot/grub2
 	ext="AA64"
 fi
 
 # we use pure BLS, so don't need grub2-mkconfig
 ostree config --repo rootfs/ostree/repo set sysroot.bootloader none
 
-# install uefi grub
-mkdir -p rootfs/boot/efi/EFI/{BOOT,fedora}
-cp "/boot/efi/EFI/BOOT/BOOT${ext}.EFI" "rootfs/boot/efi/EFI/BOOT/BOOT${ext}.EFI"
-cp "/boot/efi/EFI/fedora/grub${ext,,}.efi" "rootfs/boot/efi/EFI/BOOT/grub${ext,,}.efi"
-cat > rootfs/boot/efi/EFI/fedora/grub.cfg << 'EOF'
+if [ "$arch" == "s390x" ]; then
```

Comment:
minor nit: lets invert this to have the common case first and the s390x case second.

Reply:
Also lets add a comment here that this is a workaround until we get the zipl>grub-emu>fcos path working and link to the discussion

Reply:
Agreed.